test-gen-2/src/LightSm.csx

```diff
@@ -315,7 +322,7 @@ void PrintHtml(TextWriter writer,  string smName, string mocksCode, string merma
             button.innerText = eventName;
             button.addEventListener('click', () => {
                 clearHighlightedEdges();
-                addHistoryRow(new Date(), "Dispatched " + eventName);
+                addHistoryRow(new Date(), "Dispatched " + eventName, true);
```

Comment:
I think this is where a merge conflict will be with my latest changes. Resolve however you think is best.